packages/tsconfig/package.json

```diff
@@ -2,12 +2,8 @@
   "name": "tsconfig",
   "version": "0.0.0",
   "private": true,
-  "files": [
-    "base.json",
-    "nextjs.json",
-    "react-library.json"
-  ],
-  "scripts": {
-    "update": "pnpm update --latest && pnpm install"
+  "license": "MIT",
```

Comment:
I would like this not to have MIT licence.

Reply:
ok